index.html

```diff
@@ -959,8 +973,8 @@ <h6 class="after-cover-text subs-title mt-3 ml-3">SUBSCRIBE</h6>
                    </div>  
                 <div class="text-md-left text-sm-center text-center text-font mt-5 pt-5 mt-md-0 pt-md-0 col-md-6 col-lg-5 ml-lg-5 d-block mx-auto">
                     <h6 class="font-weight-bolder mt-4">In-depth Stories You Don't See Elsewhere About Topics You Want to</h6>
-                    <p class="pt-3 pt-md-0">All Access + Weekly Delivery.</p>
-                    <p>Choose A Membership That's Perfect for You!</p>
+                    <p class="pt-3 pt-md-0 text-muted">All Access + Weekly Delivery.</p>
+                    <p class="text-muted">Choose A Membership That's Perfect for You!</p>
                 
                     <button class="subs-btn text-white bg-danger pt-1 pb-1 pl-1 pr-1 primary-font font-weight-bolder border rounded-sm" type="submit">SUBSCRIBE ></button>
```

Comment:
Please add padding around these and align them correctly as present on the original page.